components/Common/Button/index.module.scss

```diff
@@ -0,0 +1,59 @@
+.button {
+  @apply px-[18px] py-[10px] font-semibold;
+
+  &.primary {
+    @apply bg-green-600 text-white rounded border border-green-600 shadow-sm;
+
+    &:hover {
+      @apply bg-green-700 border-green-700;
+    }
+
+    &:focus {
+      @apply bg-green-700 border-green-700;
+    }
+
+    &:disabled {
+      @apply bg-green-600 opacity-50;
+    }
+
+    &.special {
+      @apply bg-neutral-100/0 bg-gradient-to-r from-green-600/30 via-green-500/20 to-green-600/10 shadow-sm  border border-green-600/30;
+
+      &:disabled {
+        @apply opacity-50;
+      }
+    }
+  }
+
+  &.secondary {
+    @apply text-neutral-800 rounded-lg disabled:opacity-50 dark:text-neutral-200/100;
+
+    &:hover {
```

Comment:
There shouldn't be class names for states.

You should use `:focus` and `:hover` selectors.

Reply:
No. `:hover` and `:focus` pseudo selectors. 

(https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes, https://developer.mozilla.org/en-US/docs/Web/CSS/:hover)

But I just realised you are already using them. I read `&.hover` instead of `&:hover`, my bad!